docs/Concepts/Permissioning/PermissionsOverview.md

```diff
@@ -2,8 +2,8 @@
 
 GoQuorum supports two network permissioning models.
 
-* [Basic network permissioning](BasicNetworkPermissions.md): Controls which nodes can connect to a given node and also to which nodes the given node can dial out to.
-* [Enhanced network permissioning](Enhanced/EnhancedPermissionsOverview.md): Caters for enterprise-level needs by having a **smart contract-based permissioning model**. This allows for significant flexibility to manage nodes, accounts, and account-level access controls. There are two versions of enhanced network permissioning as described below:
+* **Basic network permissioning:** Controls which nodes can connect to a given node and also to which nodes the given node can dial out to. Refer [here](BasicNetworkPermissions.md) for further details.
```

Comment:
the previous version was better. link like `[here]()` or `[click this link]()` [should be avoided for multiple reasons ](https://medium.com/@heyoka/dont-use-click-here-f32f445d1021)